### ISSUE TITLE
Move Tor XPC service into separate repo

### ISSUE THREAD

**Issue by fpietrosanti** [chrisballinger/ChatSecure-Mac#1]
This ticket to split the project chatsecure-mac into a "build container" for MacOS/Tor integrated app, enabling other application to leverage the effort that's being put in place to make a Server Application to be easily deployed on Mac OS X with Tor integrated.

Most of that complexity and effort is "generic" and not specific to "chatsecure-mac" and would really benefit other application that would like to follow the same approach, shipping a secure/anonymous server-app as a easy-to-be-deployed MacOS X desktop app.

For example GlobaLeaks would be interested in following that kind of approach, but probably others :-)

**Comment by chrisballinger** [comment]
Of course! This repo is only a scratch pad and not even close to a finished product. I've made an effort to keep the XPC services separate and not dependent on each other, to allow for future usage within other applications.
